source/features/index.tsx

```diff
@@ -290,3 +290,11 @@ const features = {
 };
 
 export default features;
+
```

Comment:
I think it should be moved to https://github.com/sindresorhus/refined-github/blob/main/source/github-helpers/index.ts